.github/workflows/ci.yml

```diff
@@ -0,0 +1,71 @@
+name: CI
+
+on:
+  pull_request:
+
+jobs:
+  static-analysis-phpstan:
+    name: "Static Analysis with PHPStan"
+    runs-on: ubuntu-18.04
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.4"
+
+    steps:
+      - name: "Checkout code"
+        uses: "actions/checkout@v2"
+
+      - name: "Install PHP"
+        uses: "shivammathur/setup-php@v2"
+        with:
+          coverage: "none"
+          php-version: "${{ matrix.php-version }}"
+          extensions: mbstring
```

Comment:
is mb string needed??

Reply:
That's something I blindly copied from the master branch: https://github.com/doctrine/orm/pull/8049

Reply:
@beberlei maybe you know?

Reply:
i don't think you need it. the version of php on github runners comes with mbstring intsalled

Reply:
https://github.com/actions/virtual-environments/blob/48f3964fe67be7ba20761c4c186ba00418c3bb86/images/linux/scripts/installers/1804/php.sh#L156

Reply:
Maybe that was not the case in the past, which would explain why it was needed in the master branch. I will remove it.